app/scripts/directives/podMetrics.js

```diff
@@ -130,6 +146,18 @@ angular.module('openshiftConsole')
         // Show last hour by default.
         scope.options.timeRange = _.head(scope.options.rangeOptions);
 
+        function generateVolumeDataSets() {
+          angular.forEach(scope.pod.spec.volumes, function(volumeData) {
+            if (volumeData.persistentVolumeClaim !== undefined) {
+              volumeDataSets.push({
+                id: "filesystem/usage",
```

Comment:
Won't this return the same values for each volume if there is more than one? I don't see anything identifying the specific volume being passed to `MetricsService`.